### DISCUSSION TITLE
[Verticapy 0.12.0] Method to_sql not available for the MCA object

### DISCUSSION THREAD

**Discussion by rbensa** [vertica/VerticaPy#526] [Q&A] [answered]
Hello,

Same question and same case (processing of the "MCA" type omited) than the previous closed issue #487, I would like to know if this method will be implemented in the next versions (which one ?), please.

Kind regards,

Réda

**Comment by oualib** [comment, marked as answer]
Hi Réda,
It is not yet implemented.
Do you need the 'to_sql' method for MCA object? If yes, can you explain the use-case. It will generate the standard SQL output for the object and because of its nature, the output can result on a very long SQL code.

**Comment by rbensa** [comment]
Hello,

The purpose of my post was to find out if it was a bug or if the method was not yet implemented.

Thanks for the information.

Kind regards,

Réda
